server/common/HatoholArmPluginInterface.cc

```diff
@@ -157,6 +157,11 @@ struct HatoholArmPluginInterface::Impl {
 		connected = false;
 		hapi->onSessionChanged(NULL);
 	}
+	
+	bool getconnectedstatus(void) {
```

Comment:
I propose 'isConnected()'.
'getConnectedStatus()' make people think of getting a connected status.
And, status can be any values (2 or more) in general. However, the return type of the method is bool.